<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="CTLINDT-0919" name="test.routeone.testcase.Maxim" tests="5" failures="1" timestamp="24 Oct 2019 09:58:39 GMT" time="159.135" errors="0">
  <testcase name="invalidPasswordLogin" time="17.891" classname="test.routeone.testcase.Maxim"/>
  <testcase name="performanceUserLogin" time="45.458" classname="test.routeone.testcase.Maxim"/>
  <testcase name="problemUserLogin" time="36.629" classname="test.routeone.testcase.Maxim">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Expected condition failed: waiting for visibility of element located by By.cssSelector: [data-test=&#039;error&#039;] (tried for 20 second(s) with 500 milliseconds interval)">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Expected condition failed: waiting for visibility of element located by By.cssSelector: [data-test='error'] (tried for 20 second(s) with 500 milliseconds interval)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at dweb.aut.pages.templates.PageTemplate.waitUntilElementIsVisible(PageTemplate.java:376)
	at pages.ctis.HomePage.verifyApplicationDisplayedError(HomePage.java:61)
	at test.routeone.testcase.Maxim.problemUserLogin(Maxim.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- problemUserLogin -->
  <testcase name="standardUserLogin" time="36.967" classname="test.routeone.testcase.Maxim"/>
  <testcase name="lockedUserLogin" time="22.190" classname="test.routeone.testcase.Maxim"/>
</testsuite> <!-- test.routeone.testcase.Maxim -->
